Taskfile.yml

```diff
@@ -0,0 +1,12 @@
+# https://taskfile.dev/
+
+version: '3'
+
+tasks:
+  default:
+    cmds:
+      - NXF_VER=21.04.0-EDGE nextflow run main.nf -params-file params/params.yml -resume -with-tower
```

Comment:
Let's update this to the latest version.

Reply:
ok! :+1: